cmd/minikube/cmd/node/add.go

```diff
@@ -0,0 +1,57 @@
+package node
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"k8s.io/minikube/cmd/minikube/profile"
+	cmdutil "k8s.io/minikube/cmd/util"
+	"k8s.io/minikube/pkg/minikube"
+	cfg "k8s.io/minikube/pkg/minikube/config"
+)
+
+func NewCmdAdd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "add <node_name>",
+		Short: "Adds a node to the cluster",
+		Long:  "Adds a node tot the cluster",
```

Comment:
"to the cluster" :)

Reply:
Probably not the most tricky part about the rebase, but thanks :-)